pdb2sql/StructureSimilarity.py

```diff
@@ -299,9 +278,7 @@ def compute_irmsd_fast(self,izone=None,method='svd',cutoff=10,check=True):
         # return the RMSD
         return self.get_rmsd(xyz_contact_decoy,xyz_contact_ref)
 
-
-
-    def compute_izone(self,cutoff=5.0,save_file=True,filename=None):
+    def compute_izone(self,cutoff=10.0,save_file=True,filename=None):
```

Comment:
why a cutoff of 10 now ?

Reply:
It should be 10Å for iRMSD and 5Å for FNAT. 
It's a bug but fine, the code invoking it resets the cutoff to 10.